src/app/repositories/firebase/firebase_loader.py

```diff
@@ -22,7 +23,7 @@ def get_schemas_collection(self) -> firestore.CollectionReference:
         """
         return self.schemas_collection
 
-    def _connect_db(self) -> firestore.Client:
+    def _connect_client(self) -> firestore.Client:
```

Comment:
I tried and I couldn't get it to work :(
I've tried removing the handler. If I run all the tests locally, everything passes. However, as soon as I push it the unit tests fail in the pipeline. I have no idea why, I suspect it's python being python, which appears to run all the functionality in the Loader BEFORE the pytest mocks can patch out the dependencies.

There is possibly a way around this with more investigation, but I felt that rather than spend loads of time trying to figure it out it was better to create another ticket in the future if needed.

Reply:
Didn't notice it doesn't work on cloud. Agree if that is the case we can left it to be reviewed in the unit test review card.